### ISSUE TITLE
Gravity Forms + Custom Post Types + Repeater Field

### ISSUE THREAD

**Issue by lincolnlemos** [kodie/gravityforms-repeater#39]
There is no compability with plugin "Gravity Forms + Custom Post Types"?
I try add a new custom post type form, but the repeater fields don't save.

**Comment by knvpk** [comment]
yes, it is not saving, Anybody has an other alternative for this?
